doc/src/_static/default.css_t

```diff
@@ -290,10 +290,12 @@ p.admonition-title:after {
 }
 
 pre {
+    font-family: DejaVu Sans Mono, monospace;
+    font-size: 80%;
```

Comment:
This makes the font smaller for the majority of users who don't have DejaVu Sans Mono. I would just leave it out and let the DejaVu version be bigger.

Reply:
But in 100% it was looking really bigger so i thought to set font size to be 80%.

I had changed it to 100% now .

Reply:
It is a bigger font, but I think most people will not have it installed, so until we can get a webfont for it, we should optimize for the default monospace fonts.